build_sdk.py

```diff
@@ -102,7 +111,7 @@ class ConfigInfo:
             "KernelPrinting": True,
             "KernelVerificationBuild": False
         }
-    ),
+    )
```

Comment:
Let's avoid unrelated changes. In this case the trailing comma was intentional.

What we will do here in future is use: https://black.readthedocs.io/en/stable/index.html to auto-format all the Python code, which should avoid any of these issues.